concourse/pipelines/deploy-cloudfoundry.yml

```diff
@@ -234,118 +225,173 @@ jobs:
     serial: true
     plan:
       - aggregate:
+        - get: paas-cf
+          passed: ['generate-manifest']
+        - get: cf-manifest
+          passed: ['generate-manifest']
+          trigger: true
+        - get: bosh-secrets
+      - aggregate:
         - task: stemcell-tarball
           config:
             params:
-              STEMCELL_VERSION: {{stemcell-version}}
-            image: docker:///governmentpaas/curl-ssl
+              VERSION: {{stemcell-version}}
+              NAME: bosh-aws-xen-hvm-ubuntu-trusty-go_agent
+              URL: https://bosh.io/d/stemcells/"${NAME}"?v="${VERSION}"
+              BOSH_PASSWORD: {{bosh_password}}
+              TYPE: stemcell
+            image: docker:///concourse/bosh-deployment-resource
             platform: linux
+            inputs:
+            - name: paas-cf
+            - name: bosh-secrets
```

Comment:
Remove this as mention in https://github.com/alphagov/paas-cf/pull/82/files#r51403781